types/dispatcher.d.ts

```diff
@@ -12,6 +12,8 @@ type AbortSignal = unknown
 
 export default Dispatcher
 
+export type UndiciHeaders = IncomingHttpHeaders | string[] | Iterable<[string, string | string[] | undefined]> | null
```

Comment:
This definition should be in the docs.

Reply:
Correct it is here https://github.com/nodejs/undici/blob/main/docs/docs/api/Dispatcher.md#parameter-undiciheaders